templates/netlify/remix.init/remix.config-edge.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * @type {import('@remix-run/dev').AppConfig}
+ */
+module.exports = {
+  serverBuildTarget: "netlify-edge",
+  server: "./server.js",
+  ignoredRouteFiles: ["**/.*"],
```

Comment:
Let's keep the commented out values as well for convenience

```suggestion
  ignoredRouteFiles: ["**/.*"],
  // appDirectory: "app",
  // assetsBuildDirectory: "public/build",
  // serverBuildPath: ".netlify/functions-internal/server.js",
  // publicPath: "/build/",
```